notebooks/gotcha.clj

```diff
@@ -0,0 +1,27 @@
+(ns gotcha
+  [:require
+   [scicloj.kindly.v4.kind :as k]])
+
+(k/md "# Gotchas...
+Although we try to avoid such things, sometimes, when you're fighting the host language, it's just not practical to carry over the original conventions. Here we will try to keep an up-to-date list of possible surprises (that are not easy to 'fix').
```

Comment:
```suggestion
Although we try to avoid such things, sometimes, when you're fighting the host language, it's just not practical to carry over the original conventions. Here we will try to keep an up-to-date list of possible surprises, when coming from Mathematica to Wolframite (that are not easy to 'fix').
```

Reply:
maybe Wolfram instead of Mathematica?